### PR TITLE
Scaffold for new pages

### PR DESCRIPTION
**Pull request by ajpal**
This PR updates the route for manifest to be `edit` to follow rails best practice. It also adds the scaffolding for the two new pages we will need to enable levelbuilders to actually upload datasets.

/datasets/manifest/edit
image

/datasets/
image

/datasets/tableName
image

## Links

- [jira](https://codedotorg.atlassian.net/browse/STAR-948)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by epeach — APPROVED** [review]
Not blocking question: is there a plan for levelbuilder to be i18n-ed at some point?

**Comment by epeach** [comment]
Oh, also, as these pages are filled with content is there a way to add some sanity tests around them?

File: apps/src/sites/studio/pages/datasets/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DatasetList from '@cdo/apps/storage/levelbuilder/DatasetList';
+
+$(document).ready(function() {
```
Comment (jmkulwik) [inline apps/src/sites/studio/pages/datasets/index.js:5]: I was curious why we needed the document.ready here, so I googled it and couldn't find a clear answer besides React.render needs it. I saw some indication too that ReactDOM.render doesn't need the document.ready as it implicitly waits. Do you know if that's the case here?

**Comment by ajpal** [comment]
> Not blocking question: is there a plan for levelbuilder to be i18n-ed at some point?

There was some discussion of that on #33066. Consensus is that since there's no current work scheduled to translate LB, it makes sense to leave these strings out for now, and we can always add them later if needed.

File: apps/src/sites/studio/pages/datasets/index.js (hunk 1 above)
Comment (ajpal) [reply apps/src/sites/studio/pages/datasets/index.js:5]: I'm not exactly sure, but I just followed the model of every other page in this directory, which all have `$(document).ready`

**Comment by ajpal** [comment]
> Oh, also, as these pages are filled with content is there a way to add some sanity tests around them?

Yes, definitely! Tests are next on my list :)

**Review by jmkulwik — APPROVED** [review]

**Pull request merged**